### PR TITLE
Fixing CMR-4148 - switching the GET DATA and GET SERVICES sub types

### PR DESCRIPTION
**Pull request by eereiter**
I am fixing CMR-4148 where we need to switch the GET DATA and GET Services sub types for Related URLs.

### CODE REVIEW

File: umm-spec-lib/src/cmr/umm_spec/util.clj [hunk 1]
```diff
@@ -31,44 +31,44 @@
...
-                                  "SUBSETTER"
-                                  "THREDDS CATALOG"
-                                  "THREDDS DATA"
-                                  "THREDDS DIRECTORY"
-                                  "WEB COVERAGE SERVICE (WCS)"
-                                  "WEB FEATURE SERVICE (WFS)"
-                                  "WEB MAP FOR TIME SERIES"
-                                  "WEB MAP SERVICE (WMS)"
-                                  "WORKFLOW (SERVICE CHAIN)"]}
+  {"DistributionURL" {"GET DATA" ["DATACAST URL"
```
Comment (oubiwann) [inline umm-spec-lib/src/cmr/umm_spec/util.clj]: I'm very new to the CMR and its taxonomy ... so forgive the naïveté: it looked right to me before? 

Aren't Earthdata Search, ECHO, etc. services? And aren't OpenSearch, WCS, etc. data sources?

File: umm-spec-lib/test/cmr/umm_spec/test/validation/related_url.clj [hunk 2]
```diff
@@ -369,3 +369,5 @@
     (coll/map->UMM-C {:RelatedUrls [(merge
                                      url-type
                                      {:URL "https://www.foo.com"})]})))))
+(comment
```
Comment (laurenfrederick) [inline umm-spec-lib/test/cmr/umm_spec/test/validation/related_url.clj]: Remove this.

Comment (eereiter) [inline umm-spec-lib/test/cmr/umm_spec/test/validation/related_url.clj]: done

File: umm-spec-lib/src/cmr/umm_spec/util.clj (hunk 1 above)
Comment (oubiwann) [inline umm-spec-lib/src/cmr/umm_spec/util.clj]: Awesome -- thanks for the link. Reading now ...

**Pull request merged**